js/locale/date_format.json

```diff
@@ -41,12 +38,12 @@
     "ca": {
         "monthNameShort": "gen,febr,març,abr,maig,juny,jul,ag,set,oct,nov,des"
     },
+    "cak": {
+        "monthNameShort": "nab'ey ik',ruka'n ik',Mar,Rukaj ik',ro' ik',Ruwaq ik',Ruwuq ik',ruwaqxaq ik',rub'elej ik',rulaj ik',rujulaj ik',rukab'laj ik'"
```

Comment:
I've imported the strings, but they were too long, so I reverted back to English in the third commit.

Let me know if you prefer to have them squashed, or if you can squash them on merge.